util/timeutil/time.go

```diff
@@ -40,6 +41,15 @@ var (
 	}
 )
 
+// SetMonotonicityCheckThreshold configures the threshold below which
+// time jumps are not considered noteworthy.
+func SetMonotonicityCheckThreshold(d time.Duration) {
+	log.Infof("ignoring time backward jumps smaller than %s", d)
```

Comment:
Make this `log.V(1)`; we should try to minimize the amount of stuff that gets unconditionally logged at startup.